index.html

```diff
@@ -18,22 +18,125 @@
+          <li>html</li>
+          <li>css</li>
+          <li>javascript</li>
+        </ul>
+        <button class="btn">See Project</button>
+      </section>
+      <section class="portfolio-list-section">
+        <img src="./assets/Snapshoot Portfolio (2).png" alt="Snapshoot-1" />
+        <p class="project-title">Tonic</p>
+        <ul class="canopy-list">
+          <li id="canopy">CANOPY</li>
+          <li class="title-and-year">Back End Dev</li>
+          <li class="title-and-year">2023</li>
+        </ul>
+        <p class="project-description">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="teck-stacks">
+          <li>html</li>
+          <li>css</li>
+          <li>javascript</li>
+        </ul>
+        <button class="btn">See Project</button>
+      </section>
+      <section class="portfolio-list-section">
+        <img src="./assets/Snapshoot Portfolio (3).png" alt="Snapshoot-1" />
+        <p class="project-title">Multi-Post Stories</p>
+        <ul class="canopy-list">
+          <li id="canopy">CANOPY</li>
+          <li class="title-and-year">Back End Dev</li>
+          <li class="title-and-year">2023</li>
+        </ul>
+        <p class="project-description">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="teck-stacks">
+          <li>html</li>
+          <li>css</li>
+          <li>javascript</li>
+        </ul>
+        <button class="btn">See Project</button>
+      </section>
+      <section class="about-section">
+        <p class="about-me-text">About Myself</p>
+        <p class="about-me-description">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <p class="social-media">LET’S CONNECT</p>
+        <ul class="icons">
+          <li><img src="./assets/Vector (1).svg" alt="github" /></li>
+          <li><img src="./assets/Linkedin icon.svg" alt="linkedin" /></li>
+          <li><img src="./assets/Vector (3).svg" alt="s" /></li>
+          <li><img src="./assets/Vector.svg" alt="twitter" /></li>
+          <li><img src="./assets/Vector (2).svg" alt="m" /></li>
+        </ul>
+        <button class="btn get-my-resume">Get my resume</button>
+        <ul class="skills">
+          <li class="skill-name-and-arrow">Languages</li>
+          <img src="./assets/Frame 9.png" alt="js" />
+          <img src="./assets/Frame 10.png" alt="html" />
+          <img src="./assets/Frame 11.png" alt="css" />
+          <li class="skill-name-and-arrow">Frameworks</li>
+          <hr />
+          <li class="skill-name-and-arrow">Skills</li>
+          <hr />
+        </ul>
       </section>
```

Comment:
- [ ] I think it would be better if you add these items to your design to match as much as possible to the Figma template.

![WhatsApp Image 2023-01-14 at 11 52 01 AM](https://user-images.githubusercontent.com/105117832/212484722-671558bd-cd3e-4ef1-b0f4-8e87e0330b86.jpeg)